converter/internal/prometheusconvert/remote_write.go

```diff
@@ -4,9 +4,20 @@ import (
 	"time"
 
 	"github.com/grafana/agent/component/prometheus/remotewrite"
+	"github.com/grafana/agent/converter/internal/common"
+	"github.com/grafana/agent/pkg/river/token/builder"
 	promconfig "github.com/prometheus/prometheus/config"
 )
 
+func appendRemoteWrite(f *builder.File, promConfig *promconfig.Config) *remotewrite.Exports {
```

Comment:
I believe this pattern of translating to flow and appending to builder then returning the component exports is a good 1 for consistency and maintainability. It has been applied to all 3 component conversions.